README.md

```diff
@@ -62,17 +62,17 @@ Or with pip3:
 
     pip3 install --user pipenv
 
-
 ## Getting started
 
 The following commands should be performed only once; after creating the environment you will be connecting to the same environment each time you open a new session.
 
 Use pipenv to create a virtual environment and to install the rest of the dependencies:
 
-    pipenv --python /usr/local/bin/python3.7 install
+    pipenv --python 3 install
```

Comment:
Since we support only python 3 now, we can simplify this to:

    pipenv install